src/components/apps/toolbar/AppNavigation.js

```diff
@@ -101,41 +94,40 @@ function AppNavigation(props) {
     const [anchorEl, setAnchorEl] = useState(null);
 
     const appNavId = build(baseId, ids.APPS_NAVIGATION);
-    const allAppsCategory = getAllAppsCategory(constants.BROWSE_ALL_APPS);
+
+    const iconMap = new Map();
+    iconMap.set(constants.APPS_UNDER_DEV, <LockIcon />);
+    iconMap.set(constants.APPS_SHARED_WITH_ME, <FolderSharedIcon />);
+    iconMap.set(constants.FAV_APPS, <FavoriteIcon />);
+    iconMap.set(constants.MY_PUBLIC_APPS, <GroupWorkIcon />);
+    iconMap.set(constants.BROWSE_ALL_APPS, <AppsIcon />);
+    iconMap.set(constants.HPC, <StorageIcon />);
+
```

Comment:
Having the `icons` in the inside the `Category` object caused problems during `JSON.stringfy` and `JSON.parse`. React barked saying I cannot render object.